packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -483,8 +483,8 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
    * @param v The recovery id of the incoming ECDSA signature.
    * @param r Output value r of the ECDSA signature.
    * @param s Output value s of the ECDSA signature.
-   * @dev The v,r and s signature should be a signed message by the beneficiary
-   *      encrypting the authorized address.
+   * @dev The v,r and s signature should be signed by the authorized signer
+   *      key, with the beneficiary address as the message.
```

Comment:
Actually the RG contract address should be the message, here and below

Reply:
```suggestion
   *      key, with the ReleaseGold contract address as the message.
```